src/Task/SecurityCheckerEnlightn.php

```diff
@@ -45,6 +47,9 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments = $this->processBuilder->createArgumentsForCommand('security-checker');
         $arguments->add('security:check');
         $arguments->addOptionalArgument('%s', $config['lockfile']);
+        foreach ($config['allow-list'] as $cve) {
```

Comment:
Wouldn't it be better to 
```php
->addArgumentArray('--allow-list=%s', $config['allow-list'])
```
or is there a specific reason you want to add an optional argument?